xm-commons-metric/src/main/java/com/icthh/xm/commons/metric/MetricsConfiguration.java

```diff
@@ -66,9 +66,6 @@ public class MetricsConfiguration extends MetricsConfigurerAdapter {
     @Value("${application.kafkaMetric.metricTopics:#{null}}")
     private List<String> metricTopics;
 
-    @Value("${management.prometheus.metrics.export.enabled}")
-    private Boolean prometheusExportEnabled;
```

Comment:
Do we really want to remove this?

Reply:
yes, micrometer should collect all metrics and when we are registering dropwizard metrics manually, micrometer could collect them incorrectly. Now metrics are missing in grafana. Here i'm relying on micrometer metrics autocollection